pkg/apis/serving/v1alpha1/kfservice_framework_scikit.go

```diff
@@ -20,15 +20,20 @@ import (
 const (
 	SKLearnServingGRPCPort  = "9000"
 	SKLearnServingRestPort  = "8080"
-	SKLearnServingImageName = "animeshsingh/sklearnserver"
+	SKLearnServingImageName = "gcr.io/kfserving/sklearnserver"
```

Comment:
Is there a way we can share these defaults with the configmap's defaults, or do they have to remain decoupled since once is yaml and the other is go

Reply:
Like above, I would keep the name of keys same

e.g. SKLearnServingImageName

Reply:
I do not know if there is anyway to share them, but I made them same now.